app/stories/pages/eligibility.stories.tsx

```diff
@@ -1,14 +1,44 @@
+import { useState } from '@storybook/client-api'
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import EligibilityPage from '@pages/eligibility'
 
+import { getBackRoute, getForwardRoute } from '@utils/routing'
+
+import {
+  getEmptyMockSession,
+  getMockSessionData,
+} from '../../tests/helpers/mockData'
+
 export default {
   title: 'Pages/Eligibility',
   component: EligibilityPage,
+  argTypes: {
+    session: {
+      table: {
+        disable: true,
+      },
+    },
+  },
 } as ComponentMeta<typeof EligibilityPage>
 
-const Template: ComponentStory<typeof EligibilityPage> = () => (
-  <EligibilityPage previousRoute="review" />
-)
+const Template: ComponentStory<typeof EligibilityPage> = (args) => {
+  const [session, setSession] = useState(args.session)
+  args.session = session
+  args.setSession = setSession
+  args.backRoute = getBackRoute('/eligibility', session)
+  const reviewMode = args.reviewMode === undefined ? false : args.reviewMode
```

Comment:
Notes for reviewers: Similarly, `reviewMode` is a prop that gets passed in. In the application, `_app.tsx` is responsible for determining this (not shown in this commit). In storybook, it seemed like this was the best way to expose `reviewMode` as a control that then gets used by tho component. Is there a better/more best practice way to do this?